src/integration-test/java/uk/gov/hmcts/juror/api/moj/controller/ManagePoolControllerITest.java

```diff
@@ -1075,12 +1082,10 @@ public void test_transferJurorsToNewCourt_oneJuror_firstTransfer_happyPath() thr
         transferJurorPoolValidateNewlyCreatedJurorPool("111111111", sourcePoolNumber,
             "457230702", targetCourtLocation, targetServiceStartDate, COURT_USER);
 
-        long letterCount = certLetterRepository.count();
+        // verify no letters have been queued for bulk print
+        List<BulkPrintData> bulkPrintData = bulkPrintDataRepository.findAll();
+        assertThat(bulkPrintData.size()).isEqualTo(0);
```

Comment:
Why is this checking for 0 when previously it was checking for 1?

Reply:
its a court user so no letters should be queued - I added that logic in this MR